pkg/otlp/metrics/metrics_translator.go

```diff
@@ -802,7 +802,12 @@ func (t *Translator) MapMetrics(ctx context.Context, md pmetric.Metrics, consume
 						}
 					}
 				}
-				if t.cfg.withRemapping {
+				if t.cfg.withRenaming {
+					// Adds `otel*` system and process metrics to avoid conflicts with the Datadog Agent metrics
+					if isSystemMetric(md.Name()) {
```

Comment:
Why do we only prefix system metrics with `otel.*`?

Should we apply this prefix to all OpenTelemetry metrics, that way we can always distinguish between Datadog and OpenTelemetry metrics and avoid future conflicts.

Reply:
Make thing change in this PR would introduce breaking changes but we should revisit this decision in the future